libyul/backends/evm/EVMCodeTransform.h

```diff
@@ -49,13 +51,19 @@ class CodeTransform: public boost::static_visitor<>
 	CodeTransform(
 		yul::AbstractAssembly& _assembly,
 		solidity::assembly::AsmAnalysisInfo& _analysisInfo,
+		Block const& _block,
+		solidity::AssemblyStack::Language _language = solidity::AssemblyStack::Language::StrictAssembly,
+		bool _allowStackOpt = true,
 		bool _yul = false,
```

Comment:
This parameter is not needed anymore, it can be determined from the `_language` parameter.